### ISSUE TITLE
Thumbnails not loading in Firefox

### ISSUE THREAD

**Issue by woogygun** [blasten/turn.js#149]
Hi there for some reason the thumbnail div on magazine is not loading in firefox at all. The div loads fine in Safari and Chrome

**Comment by blasten** [comment]
Could you please send me a link to your webpage?

**Comment by ndwl** [comment]
Was experiencing the same problem, however found problem to be related to using modernizr-2.6.1.min.js instead of 2.5.3

**Comment by blasten** [comment]
@ndwl  Do you mean using an older version?

**Comment by ndwl** [comment]
No. I'm using the latest download. but when I link to modernizr-2.6.1.min.js instead of modernizr-2.5.3.min.js that is part of the package the thumbnails don't appear, when I switched back they appeared again.

**Comment by blasten** [comment]
I see. Let me check that.
